server/chat-plugins/chatlog.ts

```diff
@@ -651,40 +518,15 @@ export abstract class Searcher {
 		buf += `</div>`;
 		return LogViewer.linkify(buf);
 	}
-	async runSearch(
-		context: Chat.PageContext, search: string, roomid: RoomID, date: string | null, limit: number | null
-	) {
-		context.title = `[Search] [${roomid}] ${search}`;
-		if (!['ripgrep', 'fs'].includes(Config.chatlogreader)) {
-			throw new Error(`Config.chatlogreader must be 'fs' or 'ripgrep'.`);
-		}
-		context.setHTML(
-			`<div class="pad"><h2>Running a chatlog search for "${search}" on room ${roomid}` +
-			(date ? date !== 'all' ? `, on the date "${date}"` : ', on all dates' : '') +
-			`.</h2></div>`
-		);
-		const response = await PM.query({search, roomid, date, limit, queryType: 'search'});
-		return context.setHTML(response);
-	}
 	async runLinecountSearch(context: Chat.PageContext, roomid: RoomID, month: string, user?: ID) {
 		context.setHTML(
 			`<div class="pad"><h2>Searching linecounts on room ${roomid}${user ? ` for the user ${user}` : ''}.</h2></div>`
 		);
 		const results = await PM.query({roomid, date: month, search: user, queryType: 'linecount'});
 		context.setHTML(results);
 	}
-	async sharedBattles(userids: string[]) {
```

Comment:
Why are all the battle log stuff also gone? That needs to stay because all our log analysis scripts work on text files.